.circleci/config.yml

```diff
@@ -262,6 +269,41 @@ defaults:
 # -----------------------------------------------------------------------------------------------
 jobs:
 
+  b_windows:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - checkout
+      - run: systeminfo
+      - run:
+          name: "Preparing environment"
+          command: |
+            .\.circleci\prepare_env.ps1
+      - run:
+          name: "Running CMake"
+          command: |
+            $env:Path = "$Env:ProgramFiles\CMake\bin;$env:Path";
+            mkdir build
+            cd build
+            cmake .. -G "Visual Studio 16 2019" -DTESTS=ON -DBOOST_ROOT=C:/Libraries/boost_1_73_0
+      - run:
+          name: "Building Solidity"
+          command: |
+            $env:Path = "$Env:ProgramFiles\CMake\bin;$env:Path";
+            cmake --build build/ --config Release --parallel 3
+            Get-ChildItem -Path build/ -Recurse -Include "*.exe" | Select-Object -ExpandProperty FullName
+      - run:
+          name: "Creating release Zip file"
+          command: |
+            scripts\release.ps1
+      - store_artifacts: *artifact_solc_windows
+      - run:
```

Comment:
Should be its own job.

Reply:
I've cleaned up `b_windows` (now `b_win`) and created a `b_win_release` which is meant to run on release tag.

Reply:
I meant the tests should be a separate job.